tests/test_event_handlers.py

```diff
@@ -421,8 +421,21 @@ def test_plan_created(self, event_retrieve_mock, plan_retrieve_mock):
         plan = Plan.objects.get(stripe_id=fake_stripe_event["data"]["object"]["id"])
         self.assertEqual(plan.name, fake_stripe_event["data"]["object"]["name"])
 
+    @patch("stripe.Plan.retrieve", return_value=FAKE_PLAN)
+    @patch("stripe.Event.retrieve", return_value=FAKE_EVENT_PLAN_REQUEST_IS_OBJECT)
+    def test_plan_updated_request_object(self, event_retrieve_mock, plan_retrieve_mock):
+        plan_retrieve_mock.return_value = FAKE_EVENT_PLAN_REQUEST_IS_OBJECT["data"]["object"]
+
+        event = Event.sync_from_stripe_data(FAKE_EVENT_PLAN_REQUEST_IS_OBJECT)
```

Comment:
You could use the `_create_event()` helper as shown [here](https://github.com/jleclanche/dj-stripe/blob/1068dd06ed7762d2c382f5bf6a3ae521f798bf44/tests/test_event_handlers.py#L131). Just to simplify the event creation steps (which are usually the same).